### ISSUE TITLE
Change Simulator's Time

### ISSUE THREAD

**Issue by Giatrakis** [AvdLee/RocketSimApp#313]
I have a recommendation, but I don't know if it's possible. A great feature it will be if we can change the device time. Not just the label of the simulator, but the device time. This will be useful if we want to test a functionality that works for example in the midnight, when the day changes. Or maybe we want to show a label with text "Good morning"  until 12:00 pm and a label with text "Good afternoon" after 12:00 pm. Just an idea 🙂😉

**Comment by SergejLogis** [comment]
Bought RocketSim license yesterday. Today came here to ask for the very same feature! 😁

In our app, there are many things tied to a change of the day, and it's so inconvenient to adjust date/time of Mac in order to test features on Simulator! Leave aside the fact that changing system time affects Xcode builds, Calendar events, Slack and everything else. 🫣 

Definitely would be a **Nummer een** RocketSim feature for me! 🙏

**Comment by AvdLee** [comment]
@SergejLogis thanks for your input! Yeah, I totally see the value of this for many. I'll see what I can do! 🙏

**Comment by League2EB** [comment]
@AvdLee From 2023 to now, has it not been implemented yet?
I have subscribed for two years.

**Comment by AvdLee** [comment]
@League2EB indeed! That doesn't mean I didn't do anything or didn't make progress. However, my options are limited and the Simulator is closed source. It's very much _hacking_ and hoping to find an answer. That's how I've built the Network Speed Control last January.

Unfortunately, I have not found a way to change the Simulator time only just yet. The only way possible would be to change the macOS system time, but that's unacceptable. I simply can't build everything, although I wish!

**Comment by League2EB** [comment]
> [@League2EB](https://github.com/League2EB) indeed! That doesn't mean I didn't do anything or didn't make progress. However, my options are limited and the Simulator is closed source. It's very much _hacking_ and hoping to find an answer. That's how I've built the Network Speed Control last January.
> 
> Unfortunately, I have not found a way to change the Simulator time only just yet. The only way possible would be to change the macOS system time, but that's unacceptable. I simply can't build everything, although I wish!

Thank you for your hard work and response. Continuous improvement will be the motivation and reason for my continued subscription. Thank you.

**Comment by AvdLee** [comment]
@League2EB oh, that's already possible in RocketSim for a while!

https://docs.rocketsim.app/features/hzQMSrSga7BGWvxdNVdwYs/statusbar-appearance/3ZenpujzKpUhP4nUxjWyUN
